mir_eval/hierarchy.py

```diff
@@ -143,6 +169,69 @@ def _lca(intervals_hier, frame_size):
     return lca_matrix.tocsr()
 
 
+def _meet(intervals_hier, labels_hier, frame_size):
+    '''Compute the (sparse) least-common-ancestor (LCA) matrix for a
+    hierarchical segmentation.
+
+    For any pair of frames ``(s, t)``, the LCA is the deepest level in
+    the hierarchy such that ``(s, t)`` are contained within a single
```

Comment:
Maybe add a note here (or elsewhere, wherever you think is appropriate) that levels start from level 1 (as opposed to level 0).